### PR TITLE
7.x 4.x soe 3368 style blocks

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- See https://stanfordits.atlassian.net/browse/SOE-3368
- This adds styling to the related FAQ show-hide view so that it looks as purdy as the view on the FAQ page.
- This also adds module dependencies that were overlooked in the initial implementation. 🤦‍♀️ 

# Needed By (Date)
- Sprint end
- This is to be enabled on department sites.

# Criticality
- Fixes broken stuff

# Steps to Test
## Download and enable modules
`$ drush dl faq --destination=sites/default/modules/contrib`
Navigate to `sites/default/modules/stanford`
`$ git clone git@github.com:SU-SWS/stanford_faq.git`
`$ drush rr `
Check out this branch:
`$ git co 7.x-4.x-soe-3368-style-blocks`
Enable modules:
`$ drush en stanford_faq_related_show_hide_view stanford_faq_related_views stanford_faq_show_hide_hierarchy -y`
## Verify dependencies 
We want to verify that the dependencies are in place and that both Stanford FAQ and FAQ modules were enabled:
`$ drush pml | grep faq`

```
Other                            Frequently Asked Questions (faq)                                                           Module  Enabled        7.x-1.1
 Stanford                         Stanford FAQ (stanford_faq)                                                                Module  Enabled        7.x-4.x-dev
 Stanford                         Stanford FAQ Related Show-Hide View (stanford_faq_related_show_hide_view)                  Module  Enabled        7.x-4.0-dev
 Stanford                         Stanford FAQ related views (stanford_faq_related_views)                                    Module  Enabled        7.x-4.1-dev
 Stanford                         Stanford FAQ show/hide hierarchy (stanford_faq_show_hide_hierarchy)                        Module  Enabled        7.x-4.0-dev
```

## Verify styling
_Let me know if you need more details_
Create one or more taxonomy terms in the Stanford FAQ Categories taxonomy
Add a term reference field to the Stanford Page content type with vocabulary Stanford FAQ Categories
Create a context to display the Related FAQ block (`View: Stanford Related Show-Hide FAQs Block`) for  Node type: Stanford Pages
Tag one or more pages with FAQ terms
Create some FAQs tagged the FAQ terms
Navigate to the FAQ page at `\faq`
Navigate to the tagged Stanford Pages and 
verify the FAQ terms display the same as on the `\faq`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
PR: https://stanfordits.atlassian.net/browse/SOE-3381
https://stanfordits.atlassian.net/browse/SOE-3368

## Folks to notify
@annawatt 
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_faq_related_show_hide_view/stanford_faq_related_show_hide_view.info [hunk 1]
```diff
@@ -2,9 +2,11 @@ name = Stanford FAQ Related Show-Hide View
 description = FAQs view with bootstrap collapse (show/hide) classes. Displays FAQs related to the page.
 core = 7.x
 package = Stanford
-version = 7.x-4.x-dev
+version = 7.x-4.0-dev
```
Comment (sherakama) [inline modules/stanford_faq_related_show_hide_view/stanford_faq_related_show_hide_view.info]: These versions should be 7.x-4.3-dev. I know the other versions are in the format `7.x-4.x-dev` but we were young and foolish at the time.

Comment (minorwm) [reply modules/stanford_faq_related_show_hide_view/stanford_faq_related_show_hide_view.info]: @cjwest Please make all the versions be `7.x-4.3-dev` throughout

File: modules/stanford_faq_related_views/stanford_faq_related_views.info [hunk 2]
```diff
@@ -2,9 +2,11 @@ name = Stanford FAQ related views
 description = Displays FAQs related to the taxonomy terms on the page.
 core = 7.x
 package = Stanford
-version = 7.x-4.x-dev
+version = 7.x-4.1-dev
```
Comment (minorwm) [inline modules/stanford_faq_related_views/stanford_faq_related_views.info]: needs to be `7.x-4.3-dev`

File: modules/stanford_faq_show_hide_hierarchy/stanford_faq_show_hide_hierarchy.info [hunk 3]
```diff
@@ -2,9 +2,11 @@ name = Stanford FAQ show/hide hierarchy
 description = FAQs view with bootstrap collapse (show/hide) classes. Displays FAQs within the hierarchy related to the page.
 core = 7.x
 package = Stanford
-version = 7.x-4.x-dev
+version = 7.x-4.0-dev
```
Comment (minorwm) [inline modules/stanford_faq_show_hide_hierarchy/stanford_faq_show_hide_hierarchy.info]: needs to be `7.x-4.3-dev`

**Review by minorwm — CHANGES_REQUESTED** [review]

**Pull request merged**